website_multi_theme/static/src/xml/switch_website_menu.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Onestein
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<templates>
+
+    <t t-name="website_multi_theme.SwitchWebsiteMenu">
+        <li><a href="#" t-att-data-href="website.base_url or ''"><t t-esc="website.domain"/></a></li>
```

Comment:
I created a new field because field `domain` doesn't contain the protocol (http / https). See discussion here: https://github.com/OCA/website/pull/477#discussion_r194661289

Reply:
You can use ``href="//youwebsite.example"`` -- it will use the same protocol

Reply:
Yes but from a http website to https or the other way around can cause problems if the webserver is not correctly configured.

Reply:
True, but too specific case IMO.